geode-docs/tools_modules/geode_for_redis.html.md.erb

```diff
@@ -112,6 +112,7 @@ If the server is functioning properly, you should see a response of `PONG`.
  - RENAME <br/>
  - SADD <br/>
  - SDIFF <br/>
+ - SISMEMBERS <br/>
```

Comment:
this should be SISMEMBER; somehow an S was added to the end

Reply:
```suggestion
 - SISMEMBER <br/>
```